src/material/slider/slider.scss

```diff
@@ -22,15 +22,57 @@ $mat-slider-horizontal-margin: 8px !default;
   box-sizing: border-box;
   outline: none;
   vertical-align: middle;
-  margin: {
-    left: $mat-slider-horizontal-margin;
-    right: $mat-slider-horizontal-margin;
+  margin: 0;
+  min-width: $mat-slider-min-size;
+  -webkit-tap-highlight-color: transparent;
+
+  .mdc-slider__input {
+    pointer-events: auto;
+
+    &.mat-slider__input--no-pointer-events {
```

Comment:
Are these classes mirroring something in MDC or are they something we came up with? If it's the latter, they should follow our naming convention and not MDC's. E.g. something like `mat-mdc-slider-input-no-pointer-events`.